src/comparator.html

```diff
@@ -36,6 +42,9 @@
         </div>
       </article>
     </dialog>
+    <div id="location" class="absolute top-0">
```

Comment:
```suggestion
    <div id="location" class="z-9999 center w5">
```

I think this also works without needing the custom css. But yours is more explicit.

Reply:
Thanks for the suggestion, it does not work without the absolute but I managed to remove all except the `left: 50%` from the explicit style.

I also reduced one step the size of the `h2` components on each side since they take (IMO) too much space.